tests/common/scripts/bookinfo.txt

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/examples/bookinfo/snips.sh
+
+# Step 1: setup bookinfo & sleep pods
+kubectl label namespace default istio-injection=enabled --overwrite || true
+
+snip_start_the_application_services_2
+
+kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+kubectl apply -f samples/bookinfo/networking/destination-rule-all.yaml
+kubectl apply -f samples/sleep/sleep.yaml
```

Comment:
Why do you need the sleep sample?

Reply:
Removed, because it's not referenced anywhere.